### PR TITLE
fix: Crash when selecting Report Jira Issue in home screen

### PR DESCRIPTION
**Pull request by Balcan**
[jira issue](https://jira.dhis2.org/browse/ANDROAPP-2519)

### CODE REVIEW

File: app/src/main/java/org/dhis2/usescases/jira/JiraViewModel.kt [hunk 1]
```diff
@@ -45,6 +57,10 @@ class JiraViewModel : ViewModel(), JiraActions {
         issueService = retrofit.create<JiraIssueService>(JiraIssueService::class.java)
         prefs = preferenceProvider
```
Comment (ferdyrod) [inline app/src/main/java/org/dhis2/usescases/jira/JiraViewModel.kt]: This is a little redundant.

**Comment by JaimeToca** [comment]
@Balcan do we need to populate the jira sharepreferences for newer versions in the splash?

**Comment by Balcan** [comment]
> @Balcan do we need to populate the jira sharepreferences for newer versions in the splash?

Nope we don't need it :)
It doesn't have such an impact. They will just need to enter their jira credentials again.

File: app/src/main/java/org/dhis2/usescases/jira/JiraViewModel.kt (hunk 1 above)
Comment (Balcan) [reply app/src/main/java/org/dhis2/usescases/jira/JiraViewModel.kt]: true

**Pull request merged**